### ISSUE TITLE
Updated now an error. 

### ISSUE THREAD

**Issue by riggs498** [magicbug/Cloudlog#3155]
I just updated. I have not update in forever. Once it reloaded. I get this. (php7.4)

An uncaught Exception was encountered

Type: Error

Message: Call to undefined function lang()

Filename: /var/www/clients/client1/web21/web/application/views/user/login.php

Line Number: 36

**Comment by magicbug** [comment]
That would be the call to <?php echo lang('account_login_to_cloudlog'); ?>

Can you confirm what language you have selected? if its english i'd suggest grabbing all the files again

**Comment by emics** [comment]
the **lang()** function is in Cloudlog\system\helpers\language_helper.php did you have copy that folder?

**Comment by riggs498** [comment]
Let me try again. I used the update_cloudlog.sh script and everything looked fine. The language_helper.php file is there and updated.

**Comment by riggs498** [comment]
I performed a git clone in a different directory and moved it all over to the web directory. Still get the same error.

**Comment by riggs498** [comment]
I am just deleting the whole system and starting over

**Comment by riggs498** [comment]
I solved it. you can not upgrade from an older version. I wipe everything and started over. I could not get thing to load or merge. I am still getting different errors. I will try and figure those out later.
